heroic-core/src/main/java/com/spotify/heroic/HeroicCore.java

```diff
@@ -137,6 +140,11 @@
     static final boolean DEFAULT_DISABLE_BACKENDS = false;
     static final boolean DEFAULT_SETUP_SHELL_SERVER = true;
 
+    private static final String DEFAULT_COLLECTOR_HOST = "lightstep-satellite.services.heroic";
```

Comment:
since collector host and token are both required there's no need to have default options.